### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Welcome_Resources_ScreenReader: The word "Dialog " is not announced by JAWS.

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#3158]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**-A11y_AzureDataStudioStayHealthy;-A11ySev2;-A11yTCS;-A11yMAS;-NonBenchmark-DesktopApp;-MAS1.3.1;-SQL Azure Data Studio;-JAWS;-STP;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider
Commit: e8dc0d1
Date: 2020-05-27T17:24:57.733Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS1.3.1
Screen reader : JAWS

**Repro Steps:**
1 . Launch Azure Data Studio Insiders application.
2 . If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3 . Navigate till Resources section on Welcome screen.
4 . Launch screen reader as JAWS.
5 . Navigate to link documentation and hit enter.

**Actual:**
When the link documentation is activated and the dialog has opened the title and data are announced of the dialog but the word dialog of the control opened is not announced.
The same issue seen when get started is activated 

**Same issue not with NVDA**

**Same issue with the following screen:**
Videos pop-up

**Expected:**
The word dialog needs to be announced after the sentence "Azure data studio-insiders" followed by the rest of the data.

**User Impact**:
Screen reader users will not know a dialog has been opened.

**Attachment link for Reference:**
